doc/sym1.sgml

```diff
@@ -0,0 +1,136 @@
+<!doctype linuxdoc system>
+
+<article>
+<title>Synertek Systems Sym-1 specific information for cc65
+<author><url url="mailto:wayne@parhamdata.com" name="Wayne Parham">
+
+<abstract>
+An overview over the Sym-1 runtime system as it is implemented for the cc65 C compiler.
+</abstract>
+
+<!-- Table of contents -->
+<toc>
+
+<!-- Begin the document -->
+
+<sect>Overview<p>
+
+This file contains an overview of the Sym-1 runtime system as it comes with the cc65 C compiler.  It describes the memory layout, Sym-1 specific header files, available drivers, and any pitfalls specific to the platform.
+
+Please note that Sym-1 specific functions are just mentioned here, they are described in detail in the separate <url url="funcref.html" name="function reference">. Even functions marked as "platform dependent" may be available on more than one platform.  Please see the function reference for more information.
+
+<sect>Binary format<p>
+
+The standard binary output format generated by the linker for the Sym-1 target is a 4 kbyte machine language program.  It is, of course, possible to change this behavior by using one of the different linker configs.
```

Comment:
I think so.  Anyone that uses a SYM-1 must be familiar with machine code.  However, it might make sense here to make mention of the bin2hex program, external to cc65.  That gives a person a way to enter the program into the monitor.

Reply:
Better use `xxd` which is part of Linux and macOS.

Reply:
The Sym-1 is closely related to embedded systems, where the BIN file extension is popular.  The HEX format is also popular, so embedded developers often translate between BIN and HEX files using a converter.

Reply:
Maybe there's a misunderstanding: The question is which "tool" the user is supposed to run the binary on. Is it a real machine? Is it an emulator? Both? Based on that decision it is desirable to provide hints (aka explain very briefly) on the workflow from the ld65 output file to the running program.

Reply:
Oh, I see now Stefan and Oliver.  You're right - It would be useful to explain a little more about what to do with the BIN file and how to get it onto the Sym-1.  Or at least describe what the files are and how they relate to the Sym-1, allowing the user to choose how they want to transfer the file.

I'll write a sentence or two about that and update the sym1 document soon.